### ISSUE TITLE
Make the setup easier

### ISSUE THREAD

**Issue by bahiirwa** [klein-the-donkey/cp-admin-colors#2]
This is a great tool, however, I had a hard time navigating and administering this plugin. 

I have made a PR to improve the language.

Would it be more prudent to add the default theme setting 

![Screenshot 2019-11-05 at 15 07 18](https://user-images.githubusercontent.com/7713923/68206645-524b4d00-ffde-11e9-9cda-4c9a2e6e698a.png)

next to the profiles that are selected so that it is a simple two click step if necessary?

**Comment by klein-the-donkey** [comment]
I have seen your PR and merged it in, thank you. If you have an idea of how to do the default theme setting, I am open to hearing it.

**Comment by klein-the-donkey** [comment]
Merged the mentioned PR

**Comment by bahiirwa** [comment]
I left a little suggestion up in reporting the issue. Did you mean a PR to be evaluated?

**Comment by klein-the-donkey** [comment]
Yes, something like that. Or a code example. I currently do not have much time to research how to go about it myself.
